plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,270 @@
+---
+layout: plugins/katello/documentation
+title: Use Katello with Remote Databases
+version: nightly
+---
+
+# Setup Katello with remote databases
+
+Katello can be installed with remote databases for both postgresql and mongo.
```

Comment:
We should note for Mongo in a few places this only applies to the server and not external smart proxies.

Reply:
I think capsules could use a remote mongo (not 100% sure), but users shouldn't share their mongo db between the main server and their capsules I think.

Reply:
In conversations I've been involved, we'd been opting to support external MongoDB for the main server only. If that's changed, that's fine. Has that been tested if we want to support that?

Reply:
I can put a disclaimer at the top saying these instructions are for the main katello server only